File: app/src/main/java/com/BlizzardArmory/warcraft/Equipment/Media.java

package com.BlizzardArmory.warcraft.Equipment;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Media {

    @SerializedName("key")
    @Expose
    private Key key;
    @SerializedName("id")
    @Expose
    private int id;

    public Key getKey() {
        return key;
    }

    public void setKey(Key key) {
        this.key = key;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

}

File: app/src/main/java/com/BlizzardArmory/model/warcraft/guild/media/Media.kt
package com.BlizzardArmory.model.warcraft.guild.media

import androidx.annotation.Keep
import com.BlizzardArmory.model.common.Links
import com.google.gson.annotations.SerializedName


/**
 * The type Media.
 */
@Keep
data class Media(

    @SerializedName("_links")
    var links: Links,

    @SerializedName("assets")
    var assets: List<Asset>,

    @SerializedName("id")
    var id: Long

)